staging_docs/user/guides/02-upload.md

```diff
@@ -1,134 +1,205 @@
-        "title": "",
-        "type": "",
-        "updated_date": "",
-        "version": ""
+      "pulp_href": "/pulp/api/v3/content/rpm/advisories/018e9aec-f864-73a1-9e0b-22fc0288be75/",
+      "pulp_created": "2024-04-01T18:30:05.157638Z",
+      "pulp_last_updated": "2024-04-01T18:30:05.163938Z",
+      "id": "RHSA-XXXX:XXXX",
+      "updated_date": "2014-09-28 00:00:01",
+      "description": "",
+      "issued_date": "2014-09-24 00:00:01",
+      "fromstr": "",
+      "status": "",
+      "title": "",
+      "summary": "",
+      "version": "",
+      "type": "",
+      "severity": "",
+      "solution": "",
+      "release": "",
+      "rights": "",
+      "pushcount": "",
+      "pkglist": [],
+      "references": [],
+      "reboot_suggested": false
     }
     ```
+
+!!! note
+
+    The previous example doesn't relate the Advisory with a Repository.
+    To do so, see [Add Content to Repository](site:pulp_rpm/docs/user/guides/03-modify#add-content-to-repository).
+
+### Other Contents
+
+Rpm Content types that support individual submission are:
+`packages`,
+`advisories`,
+`modulemds`,
+`modulemd_defaults`,
+`modulemd_obsoletes`. If CLI supports it, you can use the commands below to check the necessary parameters.
+
+```bash
+pulp rpm content -t CONTENT_TYPE upload --help
+pulp rpm content -t CONTENT_TYPE create --help
+```
+
+!!! warning
+
+    `pulp-cli` may not support all of these yet, but they are available in the REST API 
+    (e.g, [modulemd_defaults POST](https://docs.pulpproject.org/pulp_rpm/restapi.html#tag/Content:-Modulemd_Defaults/operation/content_rpm_modulemd_defaults_create)).
+    
+    You may consider [opening a ticket in pulp-cli](https://github.com/pulp/pulp-cli/issues/new/choose) requesting support.
+
+    
+
+## Delete Content
+
+Removing content from Pulp entirely is slightly complicated for a number of architectural reasons.
+
+1. `repository-versions` are immutable, so you can get back to any version and have all the content available. This means that it is not possible to remove content as long as there still exists a repo-version that "knows about it".
+2. Pulp deduplicates content. Artifacts can be shared across repositories - which means the repo being looked at, may not be the only one whose repo-versions still point to the content.
+
+**The way content gets removed "from Pulp" is via the orphan cleanup task**, which will look for Artifacts that don't belong to any repository-versions, and haven't been touched in a while, and removes them.
+
+!!! warning
+    Please, follow the script steps carefully and only if you know what you are doing because you risk losing data.
+
+```bash
+#!/usr/bin/env bash
+
+# Remove content from repository
+echo "Removing content from ${REPO_NAME}"
+pulp rpm repository version destroy --repository $REPO_NAME
+
+echo "Removing orphan rpm contents"
+pulp orphan cleanup --content-hrefs $(pulp rpm content list | jq -cr '.|map(.pulp_href)') --protection-time 0
+
```

Comment:
Can we agree to stop teaching people to do the janitor's job?
Specifically `--protection-time 0` is a dangerous command in a multi user installation.

Reply:
I agree that this shouldn't generally be used, but for demonstrative purposes it's only practical.

Maybe we should include a tidbit about `--protection-time`, how it works and why it shouldn't need to be used most of the time.

Reply:
I'd say orphan cleanup should be in the admin not the user section.

Reply:
Actually, yeah, it doesn't seem like we're doing anything in the subsequent step which would require orphan cleanup.

@pedro-psb What's the purpose of the orphan cleanup here?